sand/extensions/search/wikidata_search.py

```diff
@@ -36,10 +29,9 @@ def get_class_search_params(self, search_text: str) -> Dict:
         class_params['srsearch'] = f"haswbstatement:P279 {search_text}"
         return class_params
 
-    def get_local_class_properties(self, id: str) -> Dict:
+    def get_local_class_properties(self, id: str) -> OntClass:
         """Calls local class search API to fetch all class metadata using class ID"""
-        api_data = requests.get(self.local_class_idsearch_uri + str(id))
-        return api_data.json()
+        return OntClassAR()[id]
```

Comment:
store `OntClassAR()` in self to avoid calling the function multiple times.

Reply:
added an instance variable for OntClassAR instance